### DISCUSSION TITLE
Comparison to Cython?

### DISCUSSION THREAD

**Discussion by pepijndevos** [exaloop/codon#103] [Q&A]
There are several things mentioned here https://docs.exaloop.io/codon/general/faq#how-does-codon-compare-to...

Would be interesting to see how it compares to Cython, which also statically compiles Python.

**Comment by claweyenuk** [comment]
I am working on some projects that use Cython already. Is there a good way to compare Cython with this? It looks to me like Cython errs on the side of compatibility over speed, while Codon errs on the side of speed over compatibility. Is that accurate? Are there any other big tradeoffs?

**Comment by arshajii** [comment]
FYI we just released in v0.16 a "Python extension" build mode that's similar to what Cython does (and added the description to the FAQ mentioned in the OP) -- please see https://docs.exaloop.io/codon/interoperability/pyext for more info. We plan to add a more detailed guide/tutorial soon as well.
